### PR TITLE
Es Viernes, fix image locations

### PR DESCRIPTION
**Pull request by elbruno**
Es Viernes, fix image locations and March AI Series page for Toronto Reactor

### CODE REVIEW

File: ai/22MarAIEng/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Mar 31 2022 | **[Creating and controlling a Web based video game with Computer Vision](https://www.meetup.com/Microsoft-Reactor-Toronto/events/283859052/)** <br><br>AI in video games is not new. Controlling your games with a control is old fashioned!<br>In this session we will dissect a Web browser based game created using JavaScript and HTML5, understand its basic pieces and modify it to incorporate a custom trained computer vision model to use our hand gestures as control instead of the traditional keyboard and mouse.<br><br>Presenters **Matías Iacono** is a system engineer with +20 years of experience. He has been recognized as Microsoft MVP 9 times, published 3 books and dozens of magazine articles about Microsoft technology stack. |  [MeetUp](https://www.meetup.com/Microsoft-Reactor-Toronto/events/283859052/)      | [Microsoft Reactor YouTube](https://aka.ms/Mar31WebBasedGameComputerVision)
+
+
+## Additional Resources
+
+- [Microsoft Docs- What is Custom Vision?](https://aka.ms/CustomVision-ci)
```
Comment (jimbobbennett) [inline ai/22MarAIEng/README.md:27]: Need a space after Docs

File: ai/22MarAIEng/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- [Microsoft Docs - Test and retrain a model with Custom Vision Service](https://aka.ms/CustomVisionService-ci)
+- [Microsoft Docs - Overview of image classification model by Lobe (preview)](https://docs.microsoft.com/ai-builder/lobe-overview)
+- [Microsoft Docs - Export your image classification model from Lobe to AI Builder (preview)](https://docs.microsoft.com/ai-builder/lobe-export)
+
+## Speaker and Co-Host
+
+👤 **[Bruno Capuano](http://aka.ms/elbruno)**
+
+* Blog: https://elbruno.com
```
Comment (jimbobbennett) [inline ai/22MarAIEng/README.md:36]: Maybe change to an explicit link just as elbruno.com:

[elbruno.com](https://elbruno.com)

Saves screen readers reading the https:// part

**Review by jimbobbennett — CHANGES_REQUESTED** [review]

File: ai/22MarAIEng/README.md (hunk 1 above)
Comment (elbruno) [reply ai/22MarAIEng/README.md]: changes applied !

File: ai/22MarAIEng/README.md (hunk 2 above)
Comment (elbruno) [reply ai/22MarAIEng/README.md]: changes applied !

**Pull request merged**